packages/aws-cdk-lib/aws-eks/lib/aws-auth.ts

```diff
@@ -31,6 +31,13 @@ export class AwsAuth extends Construct {
   constructor(scope: Construct, id: string, props: AwsAuthProps) {
     super(scope, id);
 
+    // Throw if the cluster does not support ConfigMap.
+    const supportConfigMap = props.cluster.authenticationMode !== AuthenticationMode.API ? true : false;
```

Comment:
if authentication mode is AuthenticationMode.API_AND_CONFIG_MAP would this be true or false ?